hack/coreapi-release-template

```diff
@@ -0,0 +1,3 @@
+COMMITTED_AT=1970-01-01 00:00:00 +0000
+COMMIT_HASH=badbadf
```

Comment:
```suggestion
COMMIT_HASH=randomhash
```

Reply:
Your suggestion is not a valid time stamp and hash.

Reply:
It is verified from smoke test

Reply:
Isn't it a placeholder..?

Reply:
when running smoke test on images build using `docker build` should pass right?
e.g. 
1. you built image using `docker build . -t foo` and deployed on your dev cluster
2. executing smoke test on the dev cluster should pass